ui/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_AUTOMAT_API_URL="http://localhost:9000"
```

Comment:
I should have added this file to `.gitignore` but will update the PR before merging.

Reply:
File deleted.